.github/workflows/test.yml

```diff
@@ -27,10 +27,10 @@ jobs:
         run: yarn build
         if: steps.cache-deps.outputs.cache-hit == 'true'
       # </common-build>
-      - name: Lint
-        run: yarn lint
       - name: Check Types
         run: yarn run check-types
+      - name: Lint
+        run: yarn lint
```

Comment:
Run lint **after** check types. It's annoying to catch type issues with a linter instead of the compiler